pypeit/core/datacube.py

```diff
@@ -1011,15 +1350,17 @@ def generate_cube_ngp(outfile, hdr, all_sci, all_ivar, all_wghts, pix_coord, bin
             pixel in the processed spec2d frames. After setting up an astropy WCS, pix_coord is
```

Comment:
See my comment above. Would it not be best to distinguish between spectral directions and spatial directions in your documentation to make things more clear.